core/README.md

```diff
@@ -1,6 +1,6 @@
 # STAC API - Core Specification
 
-- **Conformance URI:** <http://stacspec.org/spec/api/1.0.0-beta.1/core>
+- **Conformance URI:** <http://api.stacspec.org/1.0.0-beta.1/core>
```

Comment:
Should we go for https? It's the future of the Web...

Reply:
It's not working yet, which is why I held off. I was thinking of just going to https for the next version. Or if the cert is issued and https propogates we can do it.

Reply:
https is working fine for me: https://api.stacspec.org

Reply:
Just checked it and it worked for me too. I think it was propogating out. Github before wasn't letting me force https, but it just let me set it, so I did. So yes, let's switch to https. If you can do it go for it, I've got to jump on a call now, but could do it after if you don't get to it.